7024Movies/Pages/DiscoverOther.cshtml

```diff
@@ -0,0 +1,33 @@
+﻿@page
+@model _7024Movies.Pages.DiscoverOtherModel
+@{
+    var others = (DiscoverM.Result[])ViewData["Others"];
+    List<MovieId.MovieIds> otherids = (List<MovieId.MovieIds>)ViewData["OtherLkp"];
+    List<IMDB.MovieImdb> otherimdb = (List<IMDB.MovieImdb>)ViewData["OtherIMDB"];
+
+}
+
+<head>
+    <link rel="stylesheet" href="https://drvic10k.github.io/bootstrap-sortable/Contents/bootstrap-sortable.css" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.js"></script>
+    <script src="https://drvic10k.github.io/bootstrap-sortable/Scripts/bootstrap-sortable.js"></script>
+</head>
```

Comment:
Content in this page will appear in _Layout.cshtml's <body> section.  <head> cannot be a child of <body>, since <head> is a non-visible tag.  
_Layout.cshtml already has a head section.  It's best to consolidate all head entries there.